api/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cometbft/cometbft/api
 
-go 1.22
+go 1.22.5
```

Comment:
Do we have to be specific here? Can't we simply require `1.22`?

Reply:
How did the team handle previous Go version bumps? Only bumps to major releases or also minor revisions?
I thought it was the latter, because cometbft `go.mod` is currently set to the minor revision 1.22.2, thus indicating that's the minimum version we want people to compile with. 

I suppose we can set 1.22 as a minimum version, but leave Dockerfiles and CI-related things to 1.22.5 to be consistent with the `README.md` saying that we tested with 1.22.5. 

What do you think?

Reply:
Agree. Because we don't need to enforce a minor version unless we use some language features present in X or there's a CVE that is only fixed in X.

Reply:
Right. Anyway, I don't think it's our job to enforce the Go version with some fixed CVEs.